### DISCUSSION TITLE
Adaptive scale is not activated

### DISCUSSION THREAD

**Discussion by Pofig1sT161** [rustdesk/rustdesk#5671] [General]
### Bug Description

In version 1.2.0, if adaptive scale is set in the settings on both clients, it will be immediately activated upon connection. Since version 1.2.1 this has stopped working and you have to click on the adaptive scale manually when connecting.

### How to Reproduce

Clean installation on both clients and setting the settings to active adaptive scale.

### Expected Behavior

Working adaptive scale.

### Operating system(s) on local side and remote side

Windows 10

### RustDesk Version(s) on local side and remote side

1.2.1 and above

### Screenshots

Additionally, upon request, I will provide

**Comment by rustdesk** [comment]
what's adaptive scale？ screenshot please， or video

**Comment by Pofig1sT161** [comment]
![1](https://github.com/rustdesk/rustdesk/assets/103854962/02acbaa9-8bae-4e62-a414-6091fa0a8b7d)
![2](https://github.com/rustdesk/rustdesk/assets/103854962/a46d9665-5ef4-46e1-a48b-5ff3935024d7)

**Comment by Pofig1sT161** [comment]
When you first connect, the “Default display style” item set in the settings is played and shows “Original scale”, even if it is set to “Adaptive scale”.

**Comment by fufesou** [comment]
@Pofig1sT161 Hi, thanks for your feedback.

Once you have connected a peer, the peer has written the current view style to the configuration file.
Then even you change the "Default View Style", the peer configuration will keep the previous settings.
You need to change the view style in the remote toolbar menu.

Have you ever connected to the remote side before you change the default "Default View Style"?

**Comment by Pofig1sT161** [comment]
> @Pofig1sT161 Hi, thanks for your feedback.
> 
> Once you have connected a peer, the peer has written the current view style to the configuration file. Then even you change the "Default View Style", the peer configuration will keep the previous settings. You need to change the view style in the remote toolbar menu.
> 
> Have you ever connected to the remote side before you change the default "Default View Style"?


The fact of the matter is that the settings are set both on my PC and on the remote one. That is, the “Adaptive scale” mark is before my connection via RustDesk.

**Comment by fufesou** [comment]
Sorry, I cann't understand.

Can you please remove the `view_style` option in the configuration file, and try again.

The configurations on your local side is
```
%appdata%\rustdesk\config\peers\<id>.toml
```

Please upload a video to show the issue, thanks.
Or make it more clearly with come more pictures.

**Comment by youhers** [comment]
When choosing the adaptive window connection, the controlled dialog box should not have black edges, the software can automatically adjust the appropriate remote control dialog box size according to the controlled dialog box, I have always hoped to have this function, for now you need to manually click to adjust the window, hope to automatically recognize and automatically adjust the window

选择自适应窗口连接时，被控对话框的不应有黑边，软件能自动根据被控对话框调整合适的远程控制对话框大小，我也是一直希望有这个功能，就目前而言需要手动点击调节窗口，希望能自动识别并且自动调节窗口
